crates/cext/cbindgen.toml

```diff
@@ -24,8 +25,9 @@ prefix_with_name = true
 
 [export]
 prefix = "Qk"
-renaming_overrides_prefixing = false
+renaming_overrides_prefixing = true
 
 [export.rename]
 "SparseObservable" = "Obs"
 "CSparseTerm" = "ObsTerm"
+"PyObject" = "PyObject"
```

Comment:
I think you need to change `Obs` and `ObsTerm` if you change this?

Reply:
Reverted in: https://github.com/Qiskit/qiskit/pull/13934/commits/4938258dadf296019664cb2397bfecf2dd26b31b